web/webapi.yaml

```diff
@@ -64,33 +102,33 @@ components:
           type: object
           properties:
             ATIS:
-              type: string
-              description: Only set if available
-              example: "123,450 MHz"
+              type: number
+              description: Only set if available. (Format 000,000 MHz)
+              example: "123450"
```

Comment:
does this mean the unit is actually Hz? Perhaps add the unit the returned value/current example is in to the description (i read you are supposed to deliver raw values).

Reply:
I'd either use Hz as unit or add an additional field `kHz` or `MHz`.

Reply:
*Facepalm! Obviously it's in Hz..., yeah! I'll fix that! Thx!

Reply:
Be careful where you take this stuff from. :smile: `map::MapVor` uses MHz * 1000 and `map::MapNdb` kHz * 100.